package.json

```diff
@@ -7,11 +7,12 @@
   "contributors": "Gabriel Viganotti <gabrielviganotti@gmail.com>",
   "license": "(MIT OR BSD-3-Clause)",
   "scripts": {
-    "install": "node-pre-gyp install --fallback-to-build=false || neon build --release",
+    "postinstall": "if test \"$NODE_ENV\" != \"dev\" ; then node-pre-gyp install --fallback-to-build=false || neon build --release ; fi ",
```

Comment:
Hmmm, I'd set this to something other than `dev` TBH. This could mess / slow down browser builds for example, running dev. I think we need this to be specific to this repo/travis...